File: week61-70/week61/428_의상.py
def solution(clothes):
    answer = 1
    count_per_type={}
    for i in clothes:
        if count_per_type.get(i[1]):
            count_per_type[i[1]]+=1
        else:
            count_per_type[i[1]]=1
    for i in count_per_type.values():
        answer*=i+1
    return answer-1

File: week61-70/week61/428_의상.java
import java.util.*;

class Solution {
    public int solution(String[][] clothes) {
        int answer = 1;
        HashMap<String,Integer> map=new HashMap<>();
        for(String[] clothe:clothes){
            map.put(clothe[1],map.getOrDefault(clothe[1],0)+1);
        }
        for(int num:map.values()){
            answer*=num+1;
            System.out.println(num);
        }
        return answer-1;
    }
}